package/kedro_viz/server.py

```diff
@@ -539,8 +541,15 @@ def nodes_metadata(node_id):
         dataset_metadata = _get_dataset_metadata(node)
         return jsonify(dataset_metadata)
 
-    # return empty JSON for parameters type
-    return jsonify({})
+    if "parameter_name" in node:
+        # In case of 'params:' prefix
+        parameters_metadata = {
+            "parameters": {node["parameter_name"]: node["obj"].load()}
```

Comment:
Sorry could you remind me what does `node["obj"].load()` do?

Reply:
`node["obj"]` is MemoryDataSet, so it will load the value of params:key parameter (e.g. `0.2`)